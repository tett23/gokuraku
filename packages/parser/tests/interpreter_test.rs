extern crate parser;

use insta::{assert_debug_snapshot, glob};
use std::fs;

#[test]
fn run_test() {
    glob!("../fixtures/pds", "*.pds", |path| {
        let input = fs::read_to_string(path).unwrap();
        let value = parser::prose_down_script_run(&input).unwrap();

        assert_debug_snapshot!(value);
    });
}
