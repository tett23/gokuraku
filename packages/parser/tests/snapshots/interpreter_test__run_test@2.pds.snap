---
source: packages/parser/tests/interpreter_test.rs
expression: value
input_file: packages/parser/fixtures/pds/2.pds
---
()
